src/storage/files.rs

```diff
@@ -108,7 +108,7 @@ impl File<'_> {
     /// Returns the latest modification date in UTC.
     pub fn modification_date(&self) -> DateTime<Utc> {
         let epoch = unsafe { (*self.inner).modificationdate };
-        Utc.timestamp(epoch, 0)
+        Utc.timestamp_opt(epoch, 0).unwrap()
     }
```

Comment:
Wouldn't be better to return the `DateTime<Utc>` wrapped inside `Option` so we avoid `unwrap`?

Reply:
Yes, it would be, however, I wanted to do as little as possible changes to remove the deprecation. Chrono is doing `unwrap()` anyway.